src/test/kotlin/org/rustSlowTests/cargo/runconfig/RunConfigurationTestBase.kt

```diff
@@ -97,6 +93,12 @@ abstract class RunConfigurationTestBase : RsWithToolchainTestBase() {
         return CargoBuildManager.build(buildConfiguration).get()
     }
 
+    protected fun checkProject(useClippy: Boolean = false): CargoBuildResult {
+        val checkConfiguration = createBuildConfiguration()
+        checkConfiguration.configuration.command = if (useClippy) "clippy" else "check"
+        return CargoBuildManager.build(checkConfiguration).get()
+    }
```

Comment:
This method looks very similar with `buildProject`. Can we unify them? For example, pass command name to `buildProject` method with some default value (`run` for example)